x-pack/plugins/ml/public/application/explorer/explorer_charts/explorer_charts_error_callouts.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiCallOut, EuiSpacer } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import React, { FC } from 'react';
+import { ExplorerChartSeriesErrorMessages } from './explorer_charts_container_service';
+
+interface ExplorerChartsErrorCalloutsProps {
+  errorMessagesByType: ExplorerChartSeriesErrorMessages;
+}
+
+export const ExplorerChartsErrorCallOuts: FC<ExplorerChartsErrorCalloutsProps> = ({
+  errorMessagesByType,
+}) => {
+  if (!errorMessagesByType || Object.keys(errorMessagesByType).length === 0) return null;
+  const content = Object.keys(errorMessagesByType).map((errorType) => (
+    <EuiCallOut color={'warning'} size="s">
+      <FormattedMessage
+        id="xpack.ml.explorerCharts.errorCallOutMessage"
+        defaultMessage="You can't view anomaly records for {jobs} because {reason}."
```

Comment:
Swap the word `records` for `charts`.

Reply:
Updated here https://github.com/elastic/kibana/pull/84594/commits/ce964f5f6b9509419fb456c7554283d8c5a81b79